api/pkg/s3/bucketload.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3
+
+import (
+	"errors"
+	"net/http"
+
+	"encoding/xml"
+
+	"github.com/emicklei/go-restful"
+	"github.com/micro/go-log"
+	. "github.com/opensds/multi-cloud/s3/pkg/exception"
+	"github.com/opensds/multi-cloud/s3/pkg/model"
+	"golang.org/x/net/context"
+	pb "github.com/opensds/multi-cloud/s3/proto"
+)
+
+func (s *APIService) BucketLoad(request *restful.Request, response *restful.Response) {
```

Comment:
will this be integrated with stackstorm as an action ? to support the analytics workflow ?

Reply:
Yes, it's just an action in stacktorm, and it is integrated yet, see [https://github.com/opensds/orchestration/pull/89](url) please